src/test/java/com/aws/greengrass/secretmanager/SecretManagerTest.java

```diff
@@ -127,23 +138,30 @@ class SecretManagerTest {
     @Captor
     ArgumentCaptor<software.amazon.awssdk.services.secretsmanager.model.GetSecretValueRequest> awsClientRequestCaptor;
 
-    private List<SecretConfiguration> getMockSecrets() {
+    private void getMockSecrets() throws JsonProcessingException, UnsupportedInputTypeException {
```

Comment:
super-nit: since these methods return void and are no longer a _getter_ , I would call these e.g. `loadMockSecrets` instead